doc/api_assets/style.css

```diff
@@ -972,6 +973,14 @@ kbd {
   }
 }
 
+@media only screen and (max-width: 600px) {
+  .js-flavor-selector {
+    float: none;
+    display: block;
+    margin: 0 0 1em 0;
```

Comment:
that will just make it float to the right instead of the left
<img src="https://user-images.githubusercontent.com/8221854/174556736-f3853a4d-305c-4c82-9b7f-dd5cb64b46f9.png" width="300"/>

Reply:
Ok then. I'm no big designer :)